app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,31 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :hostname
+
+  def self.to_calendar(reservation)
+    ReservationCalendar.new(reservation).as_ical
+  end
+
+  def initialize(reservation, hostname)
+    super(reservation)
+    @hostname = hostname
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
+      e.dtend = Icalendar::Values::DateTime.new(reserve_end_at)
+      e.summary = "#{facility.abbreviation}: #{product.name}"
+      e.description = "#{facility.name} reservation for #{product.name}. Order number #{order.id}"
+      e.location = facility.name
+      e.url = Icalendar::Values::Uri.new(
+        Rails.application.routes.url_helpers.order_order_detail_reservation_url(
```

Comment:
I would link to the `/orders/XXX/order_details/XXX` route rather than the reservation. It has a little more information and can easily link into the reservations as well as the receipt. (Also, I notice there's no way to get from the reservation view back to the order detail view, which is not ideal, but that's how it is right now).